Pages/1-intro.md

```diff
@@ -18,12 +125,37 @@ java version "1.8.0_40"
 Java(TM) SE Runtime Environment (build 1.8.0_40-b27)
 Java HotSpot(TM) 64-Bit Server VM (build 25.40-b25, mixed mode)
 ```
+if not follow this [Install Java](https://www.java.com/en/download/help/download_options.xml)
+
+
+## Installing Leiningen
+
+Multiple paths depending on your system type.
+
+But before we go there, a brief overview of what leiningen is up to:
+
+  - it (maybe) bootstraps itself
+  - it does this by going off and downloading packages from package repositories on the internet
+  - it maintains these packages in a maven repository on your local machine (typically find this in your home directory in a directory called '.m2')
+  - eventually you might use lein to manage your own projects, at which point it will get packages you need in addition to what it needs to bootstrap
+  
+So, during bootstrapping, you will likely see it doing this sort of stuff.  Relax, that's what it's supposed to do.
 
-[Install Java](https://www.java.com/en/download/help/download_options.xml)
+For Mac people, you can install this with `brew` on a mac.
 
+Linux distros may also have Leiningen included in it's package manager.  If so, that is a very easy place to get it as well.
+
+Failing that visit the following site [Leiningen](http://leiningen.org/). Follow the instruction on the page and it is super easy!
+
+So whether via brew, package manager, shell script or .bat file from the leiningen site you are properly setup with with leiningen if you can do the following from the command line:
+
+```
+▶ lein version
+Leiningen 2.7.0 on Java 1.8.0_102 OpenJDK 64-Bit Server VM
+```
+or something similar
 
-## Installing Lein
```

Comment:
I think paragraph starting with "Failing that..." ends up being the old procedure